docs/docsite/rst/porting_guides/porting_guide_2.9.rst

```diff
@@ -68,6 +68,10 @@ The following modules will be removed in Ansible 2.13. Please update update your
  
 * vyos_l3_interface use :ref:`vyos_l3_interfaces <vyos_l3_interfaces_module>` instead.
 
+* nxos_vlan use :ref:`nxos_vlans <nxos_vlans_module>` instead.
```

Comment:
Does this module depends on nxos_vlan? I'm not sure why this phrase is here. Thanks!

Reply:
No this module doesn't depend on nxos_vlan. I missed adding vlan deprecation when I opened a PR nxos_vlans. Since it's already been merged, I have added the deprecation for that in this PR.